packages/mui-base/package.json

```diff
@@ -43,7 +43,7 @@
     "@babel/runtime": "^7.25.0",
     "@floating-ui/react-dom": "^2.1.1",
     "@mui/types": "workspace:^",
-    "@mui/utils": "workspace:*",
+    "@mui/utils": "workspace:^",
```

Comment:
Was this intentional change?

Reply:
Yes, it was done to fix the prerelease issue https://github.com/mui/material-ui/pull/42768

Reply:
Now that we are releasing the stable version, it should be reverted.